app/models/occupation_standard.rb

```diff
@@ -12,6 +12,28 @@ class OccupationStandard < ApplicationRecord
 
   validates :title, presence: true
 
+  def unregistered_clone(creator_id:, organization_id:)
```

Comment:
Little things, but would it make sense for this to have a bang, `#unregistered_clone!(...)`? Can we try a more active-reading version of this, too? `unregistered_clone` feels like a method that's getting information, not creating an object. `#clone_as_unregistered!` is more wordy but feels active. Or just `#clone!` with an implied `UnregisteredStandard` type?

Reply:
Yes, great suggestions!  I was having a hard time with the naming since `clone` is already a method.  I will make those changes.